api/controllers/userController.js

```diff
@@ -1,71 +1,58 @@
-const User = require('../models/user.js');
-const uuid = require('uuid');
+const { models } = require('../db');
 
-exports.create = (req, res) => {
-    if (!req.body) {
-        return res.status(400).send({
-            message: 'User content can not be empty!'
-        });
+async function create(req, res) {
+    try {
+        await models.user.create(req.body);
+        res.status(201).end();
```

Comment:
Would be great to return user's Id here.

Reply:
the code was modified to fetch user's id after creating